salt/utils/event.py

```diff
@@ -427,7 +427,7 @@ def connect_pull(self, timeout=1):
                     self.pusher.connect(timeout=timeout)
                     self.cpush = True
                 except Exception as exc:  # pylint: disable=broad-except
-                    log.error(
+                    log.debug(
```

Comment:
We should only log to debug the exceptions we absolutely know are okay, in this case it's StreamClosed error. Anything else (anything we don't expect) should still throw an error.

Reply:
Added a specific check for `salt.ext.tornado.iostream.StreamClosedError`